### PR TITLE
Document `tabs` shortcode

### PR DESCRIPTION
**Pull request by jdbaldry**
Screenshot 2024-07-24 at 13 24 31

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -907,6 +907,46 @@ You should generally avoid using this shortcode because every documentation page
...
+## Tabs
+
+The `tabs` shortcode creates generic tabbed content.
+The website saves the selected tab to browser storage and persists it across navigation.
+
+You create a tab using the `tab-content` shortcode within the `tabs` shortcode.
+The inner of the `tab-content` can be any Markdown.
+
+{{< admonition type="note" >}}
+You can't nest the `tabs` shortcode in a `tab-content` shortcode but you can use the `code` shortcode instead.
```
Comment (imatwawana) [inline docs/sources/write/shortcodes/index.md:919]: Maybe add an example of this as a secondary example. I don't really know what the use case is here, so I'm not immediately grokking why this guidance is here.

[the remark this answers is not in the document]
Comment (clayton-cornell) [reply docs/sources/write/shortcodes/index.md:919]: You can't define text tabs within text tabs... you can use code tabs in text tabs.

[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/write/shortcodes/index.md:919]: I'm just trying to imagine what that would look like and I can't, so I'm trying to think of why someone would try this. I'll test it out with a code shortcode later.

[the remark this answers is not in the document]
Comment (heitortsergent) [reply docs/sources/write/shortcodes/index.md:919]:
```suggestion
You can nest a `code` shortcode inside of a `tab-content` shortcode, but you can't nest a `tabs` shortcode inside of a `tab-content` shortcode.
```

I think changing the order of the sentences might make this easier to understand. Wdyt? 🤔

[the remark this answers is not in the document]
Comment (jdbaldry) [reply docs/sources/write/shortcodes/index.md:919]: Yeah much clearer!

**Comment by grafsean** [comment]
Nice feature @jdbaldry. I tried it for code as suggested. Is there a recommended way to nest code and not have as much padding?
 
Screenshot 2024-07-31 at 09 58 40

**Comment by jdbaldry** [comment]
Do you have a link to a branch or PR that I can investigate?

**Comment by grafsean** [comment]
Sent you a file in Slack.

**Pull request merged**